rtf

```diff
@@ -272,6 +278,20 @@ class Platform:
           host_list.append(hostname)
       host_type.status(self.name, host_list)
 
+  def get_server_info(self):
```

Comment:
what do you think of "get_server_token" ?  get_server_api_info ? 

server_info is is too general too me :(